tests/zfs-tests/tests/functional/removal/removal.kshlib

```diff
@@ -34,7 +34,7 @@ function wait_for_removal # pool
 	# The pool state changes before the TXG finishes syncing; wait for
 	# the removal to be completed on disk.
 	#
-	sync_pool $pool
+	sync_all_pools
```

Comment:
This should be fine, but it looks like it didn't need to be updated.

Reply:
Yeah, I was fixing a use of `sync` in that file and auto-piloted it. I've staged reverting it back for after the tests complete.

Reply:
Just FYI, I was looking in to another failures and opened 12897 which will have a minor (easily resolved) conflict with this  PR.